neo/Prompt/Utils.py

```diff
@@ -131,6 +131,38 @@ def get_from_addr(params):
     return params, from_addr
 
 
+def get_change_addr(params):
+    to_remove = []
+    change_addr = None
+    for item in params:
+        if '--change-addr=' in item:
+            to_remove.append(item)
+            try:
+                change_addr = item.replace('--change-addr=', '')
+            except Exception as e:
+                pass
+    for item in to_remove:
+        params.remove(item)
+
+    return params, change_addr
+
+
+def get_outgoing(params):
```

Comment:
same comments as for `get_change_addr`
1) if we only return 1 `outgoing` we don't need to push 1 item in a list. 
2) try to make the exception specific

Reply:
I think I addressed this issue.